src/message_interface/message_handler.rs

```diff
@@ -311,6 +310,14 @@ impl WalletMessageHandler {
                 })
                 .await
             }
+            #[cfg(feature = "events")]
+            Message::ClearListeners(events) => {
```

Comment:
I'm not sure this should be a message, `clear_listeners` should be called from the bindings.